libs/chrome-tools/background.js

```diff
@@ -0,0 +1,37 @@
+try {
+  importScripts('./sdk/rxjs-7.5.2.umd.min.js', './sdk/nuclia-sdk.umd.min.js');
+} catch (e) {
+  console.error(e);
+}
+
+chrome.runtime.onInstalled.addListener(() => {
+  chrome.contextMenus.create({
+    id: 'NucliaContextMenu',
+    title: 'Upload to Nuclia',
+    targetUrlPatterns: ['https://*/*'],
+    contexts: ['link'],
+  });
+});
+
+chrome.contextMenus.onClicked.addListener((info) => {
+  chrome.storage.local.get(['kb', 'key'], ({ kb, key }) => {
```

Comment:
To prevent any conflict in the storage, we should maybe prefix the storage keys by `NUCLIA_` or something, don't you think?

Reply:
Fixed